docs/.vuepress/config.js

```diff
@@ -26,12 +26,13 @@ module.exports = {
         children: [
           "/gaia/installation",
           "/gaia/join-testnet",
+          "/gaia/validators/validator-setup",
```

Comment:
shouldn't the validator info be in together, maybe move networks up?

Reply:
I changed it cause the end of `join-testnet` points to `validator-setup`, and because the next logical step after running a full node is to upgrade to a validator. I'll move networks down instead